hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -602,6 +603,10 @@ void Vgen::emit(const andli& i) {
   a->And(W(i.d), W(i.s1), MSKTOP(i.s0.l()), SetFlags);
 }
 
+void Vgen::emit(const testbi& i) {
+  a->Tst(W(i.s1), MSKTOP(i.s0.l()));
+}
```

Comment:
Replying to @mxw above, I think the whole point of MSKTOP as a function is to have an assertx.  The right  shifting is just replicating the sign bit.  The conditional is just checking
to make sure there wasn't anything specified in the upper bits.

The usual immediate operand to an AND instruction is unsigned.  So, in this case we could
do the same kind of function and check a different set of bits.  I'm not convinced it would
buy all that much.

I'll push the i.s0.bu() change so we can declare victory and move on to the next one.